### ISSUE TITLE
Blind users cannot use this app because it is not accessible with native screen readers such as VoiceOver.

### ISSUE THREAD

**Issue by AnonymousAccessibilityAdvocate** [cake-tech/cake_wallet#3406]
All buttons in Cake Wallet need to be labeled. Open Cake wallet, navigate with the native iOS screenreader (VoiceOver) and you'd expect to hear, "send", "receive", "swap," and so on. Instead the screenreader only reads, "button", "button", "button". This affects all iOSversions.

**Comment by MrCyjaneK** [comment]
#3402

**Comment by AnonymousAccessibilityAdvocate** [comment]
I think that the buttons need to be wrappedin a Semantics widget and setting the label property, or by using the semanticLabel property directly on widgets like.
